lib/generator/files/exercise_files.rb

```diff
@@ -0,0 +1,83 @@
+require 'erb'
+
+module Generator
+  module Files
+    module GeneratorCases
+      def self.available(track_path)
```

Comment:
The style guide seems to suggest `module_function` rather than `extend_self` so I'll use `module_function` instead: https://github.com/bbatsov/ruby-style-guide#module-function